### PR TITLE
Fixed a bug where gradle build would fail on windows.

### PR DESCRIPTION
**Pull request by Andrei94**
Gradle build would fail because of differences of path separators between linux and windows.

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@Andrei94 Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-framework&pullRequestId=1358)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-projects/spring-framework&pullRequestId=1358) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@Andrei94 Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-framework&pullRequestId=1358)!

**Comment by Andrei94** [comment]
@snicoll review please? :)

File: build.gradle [hunk 1]
```diff
@@ -1371,14 +1371,14 @@ configure(rootProject) {
 			def Properties schemas = new Properties();
 
 			subproject.sourceSets.main.resources.find {
-				it.path.endsWith("META-INF/spring.schemas")
+				it.path.endsWith("META-INF\\spring.schemas")
```
Comment (snicoll) [inline build.gradle]: That most certainly broke non-Windows build.

File: build.gradle [hunk 2]
```diff
@@ -1371,14 +1371,14 @@ configure(rootProject) {
...
 			}?.withInputStream { schemas.load(it) }
 
 			for (def key : schemas.keySet()) {
 				def shortName = key.replaceAll(/http.*schema.(.*).spring-.*/, '$1')
 				assert shortName != key
 				File xsdFile = subproject.sourceSets.main.resources.find {
-					it.path.endsWith(schemas.get(key))
+					it.path.endsWith(schemas.get(key).replaceAll('\\/','\\\\'))
```
Comment (snicoll) [inline build.gradle]: Ditto.

**Review by snicoll — CHANGES_REQUESTED** [review]
There must be a way to define those separator in a non-OS sensible ways. 

Perhaps @sdeleuze knows?

**Comment by Andrei94** [comment]
@snicoll changes made :smiley:

**Comment by snicoll** [comment]
@jhoeller you are on Windows right?  Can you check this one please?

**Comment by Andrei94** [comment]
@jhoeller any news about this PR?

**Pull request closed without merging**